src/OpenTelemetry/Trace/DefaultTracing.cs

```diff
@@ -16,49 +16,33 @@
 
 namespace OpenTelemetry.Trace
 {
-    using System.Runtime.CompilerServices;
     using OpenTelemetry.Internal;
     using OpenTelemetry.Trace.Config;
     using OpenTelemetry.Trace.Export;
     using OpenTelemetry.Trace.Internal;
 
     /// <summary>
-    /// Class that manages a global instance of the <see cref="Tracer"/>.
+    /// Class that defines the default global instances of the <see cref="Tracing"/> APIs.
     /// </summary>
-    public sealed class Tracing
+    public static class DefaultTracing
     {
-        private static Tracing tracingValue = new Tracing();
-        private static Tracer tracer;
-
-        internal Tracing()
+        /// <summary>
+        /// Initializes the default tracing implementation.
+        /// </summary>
+        public static void Init()
```

Comment:
There is **proposed** [global tracing rfc](https://github.com/open-telemetry/oteps/blob/e4ebf33440b31a851fa11c294151c2da00109758/text/0005-global-init.md) that suggests that `Init` may take `tracer` as an argument to support testing and non-global tracers better

Reply:
I've added `Init` on abstractions that take `ITracer and ISpanExporter`. The proposed RFC mentions we should allow init once and log after that. Past experience has showed me that allowing the last `Init` call win (while logging the odd behavior as a warning) is a better approach though. One of the reasons for that is testing, where we can 're-initialize' the SDK multiple times and cover the static API usage in some tests. That can be offered via `internal/InternalVisibleTo` also though.

Please note there are two `Init` methods. One on `Abstractions` and one on the implementation which uses the first.